dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -1323,9 +1326,8 @@ CDefaultContextMenu::InvokePidl(LPCMINVOKECOMMANDINFOEX lpcmi, LPCITEMIDLIST pid
             *wszDir = UNICODE_NULL;
     }
 
-    SHELLEXECUTEINFOW sei;
-    ZeroMemory(&sei, sizeof(sei));
-    sei.cbSize = sizeof(sei);
+    CComHeapPtr<WCHAR> pszParamsW;
```

Comment:
or "pszParams" as it's named in CShellLink::DoOpen() (see the other file).

Reply:
This is only used when a call to SHCloneStrAtoW is required. I'm just being clear about why this buffer exists, it's the wide version when the caller only gave us a narrow string.